pkgs/servers/mastodon/source.nix

```diff
@@ -0,0 +1,14 @@
+{ fetchgit, runCommand, jq }: let
+  src = fetchgit {
+    url = "https://github.com/tootsuite/mastodon.git";
+    rev = "v3.0.1";
+    sha256 = "11vljzarhgs6hik6qhimlaimqm5ysljf9jm402r5fpy6axh6yksc";
+  };
+  version = import ./version.nix;
+in runCommand "mastodon-source" { buildInputs = [ jq ]; } ''
```

Comment:
btw there is `applyPatches` function for that

Reply:
or even `postFetch` parameter of `fetchgit`

Reply:
Thanks for the suggestions. I reworked it to use `applyPatches`.